VERSION

```diff
@@ -1 +1 @@
-0.2.16
+0.2.17-dev
```

Comment:
please remember to increase to 0.3.x once we have the planned breaking change in review..